pkg/build/controller/config_controller.go

```diff
@@ -10,8 +10,18 @@ import (
 
 	buildapi "github.com/openshift/origin/pkg/build/api"
 	buildclient "github.com/openshift/origin/pkg/build/client"
+	buildgenerator "github.com/openshift/origin/pkg/build/generator"
 )
 
+// ConfigControllerFatalError represents a fatal error while generating a build
+type ConfigControllerFatalError struct {
+	Reason string
+}
+
+func (e ConfigControllerFatalError) Error() string {
```

Comment:
godoc this as it is exported, also explain in godoc what does the FatalError means